write-fonts/src/tables/layout.rs

```diff
@@ -467,6 +467,10 @@ impl FromIterator<(GlyphId, u16)> for ClassDef {
 
 impl ClassDefBuilder {
     fn prefer_format_1(&self) -> bool {
+        // empty table must always use format 2
```

Comment:
I would change the comment saying it's more compact to use format 2. Or am I missing something?

Reply:
oops yes I was confused by the fact that format 1 has a `startGlyphId` field, and was thinking that that meant it had to have at least one item in it.

In addition in the process of going and looking through this again I noticed another miscalculation here, and have pushed an additional commit to fix that.